documentation/FlowManager.md

```diff
@@ -60,26 +60,67 @@ Note that a separate logout of "reset" is not necessary. The final condition of
 ### User Login (GET /user/:gpiiKey/login)
 
 * **description**: Log in a user to the system
-* **Supported modes**: works only with a locally installed GPII framework (i.e. non-cloud-based flowmanager)
 * **route:** `/user/:gpiiKey/login` where :gpiiKey should be the GPII key of the user
 * **method:** `GET`
 * **return:** Message saying that user successfully logged into the system or an error message.
 
 ### User Logout (GET /user/:gpiiKey/logout)
 
 * **description**: Log out a user of the system
-* **Supported modes**: works only with a locally installed GPII framework (i.e. non-cloud-based flowmanager)
 * **route:** `/user/:gpiiKey/logout` where `:gpiiKey` should be the GPII key of the user
 * **method:** `GET`
 * **return:** Message saying that user successfully logged out of the system or an error message.
 
+## APIs on Cloud Based Flow Manager
+
+### Check the readiness of Cloud Based Flow Manager (GET /ready)
+
+* **description**: Check whether Cloud Based Flow Manager is up and running.
+* **route:** `/ready`
+* **method:** `GET`
+* **return:** Return http status code 200 when Cloud Based Flow Manager is up and running. Otherwise, return http status
+ code 500.
+
+### Check the liveness of Cloud Based Flow Manager (GET /health)
+
+* **description**: Check whether Cloud Based Flow Manager is able to handle requests.
+  * When Cloud Based Flow Manager and Preferences Server are running together as one server, the liveness endpoint
+    checks the database connection.
+  * When Cloud Based Flow Manager and Preferences Server are running as separate servers, the liveness endpoint
+    checks the communication with Preferences Server.
+* **route:** `/health`
+* **method:** `GET`
+* **return:** Return http status code 200 when Cloud Based Flow Manager is up and running. Otherwise, return http status
```

Comment:
Should this say "is up and running" again? Maybe it should say "is able to handle requests"?

As above, I think it might help to say more about what this means.